documentation/modules/proc-upgrading-the-cluster-operator.adoc

```diff
@@ -9,7 +9,8 @@ This procedure describes how to upgrade a Cluster Operator deployment to a later
 
 .Prerequisites
 
-* An existing Cluster Operator deployment.
+* An existing Cluster Operator deployment is available.
+* The installation files have been xref:downloads-{context}[downloaded for the new version].
```

Comment:
Consider the active voice:

* You have downloaded the [installation files](#) for the new version.